# 
# @brief    : VISION_POSITION
# @time     : 2018/09/26
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
#

# position gotten from vision

float32 x_deep          #point X position, in m
float32 y_deep          #point Y position, in m
float32 z_deep          #point Z position, in m

float32 x_dis          #point X position, in m
float32 y_dis          #point Y position, in m
float32 z_dis          #point Z position, in m

